lib/sequelize.js

```diff
@@ -1200,8 +1197,8 @@ class Sequelize {
       options = last;
 
       // remove options from set of logged arguments if options.logging is equal to console.log
```

Comment:
If we allow console.debug we should also mention that here

Reply:
Done!